app/browser/api/ledger.js

```diff
@@ -2241,15 +2241,39 @@ const deleteSynopsis = () => {
 }
 
 let newClient = null
-const transitionWalletToBat = (state) => {
+const transitionWalletToBat = () => {
   let newPaymentId, result
+  const newClientPath = 'ledger-newstate.json'
```

Comment:
nitpick: we should probably move all constants like this to the top of the file.  For example, see `statePath`